js/todoRender.js

```diff
@@ -0,0 +1,37 @@
+const todoTemplate = ({ id, contents, completed }) => {
+    return `<li data-id=${id} class=${completed? 'completed':''}>
+            <div class="view">
+                <input class="toggle" type="checkbox" ${completed? 'checked' : ''}/>
+                <label class="label">${contents}</label>
+                <button class="destroy"></button>
+            </div>
+        <input class="edit" value='${contents}' />
+    </li>`;
+};
+
+export const todoListRender = () => {
+    const $todoList = document.querySelector('.todo-list');
+
+    // 다시 고려해볼 것
+    const $todoApp = document.querySelector(".todoapp");
+    const $todoCount = $todoApp.querySelector(".todo-count > strong");
+
+    let todoArr = [];
+    for(let i = 0; i < localStorage.length; i++){
+        let key = localStorage.key(i);
+        let todoItem = JSON.parse(localStorage.getItem(key));
+        todoItem.id = key;
+        todoArr.push(todoItem);
+        // console.log(todoItem.contents);
+    }
```

Comment:
```suggestion
  const todoArr = Object.entries(localStorage).map(([key, val]) => ({
    ...JSON.parse(val),
    id: key,
  }));
```

for문으로 사용된부분을, 공통 피드백 내용과같이 위와같이 map으로 바꿔주시면 가독성이 더욱 좋아질 것 같습니다!